.github/main.go

```diff
@@ -171,24 +171,12 @@ func (ci *CI) withTestWorkflows(runner *dagger.Gha, name string) *CI {
 		WithJob(runner.Job("engine-test-publish", "engine publish --image=dagger-engine.dev --tag=main --dry-run")).
 		WithJob(runner.Job("scan-engine", "engine scan")).
 		With(splitTests(runner, "test-", false, []testSplit{
-			{"modules", []string{"TestModule"}, &dagger.GhaJobOpts{
-				Runner: []string{GoldRunner(false)},
-			}},
-			{"module-runtimes", []string{"TestGo", "TestPython", "TestTypescript", "TestElixir", "TestPHP", "TestJava"}, &dagger.GhaJobOpts{
-				Runner: []string{GoldRunner(false)},
-			}},
-			{"cli-engine", []string{"TestCLI", "TestEngine"}, &dagger.GhaJobOpts{
-				Runner: []string{GoldRunner(false)},
-			}},
 			{"cgroupsv2", []string{"TestProvision", "TestTelemetry"}, &dagger.GhaJobOpts{
 				// HACK: our main runners don't support cgroupsv2
 				Runner: []string{"ubuntu-latest"},
 				// NOTE: needed to silence redis warning logs in tests
 				SetupCommands: []string{"sudo sysctl -w vm.overcommit_memory=1"},
 			}},
```

Comment:
We can move this into `testdev` as well.

It doesn't need a stable engine, but what's happening now is that `evenything-else` in `testdev` is picking up `TestProvision` (since it's a test suite that's remaining, see the logic in `splitTests`)

Reply:
I understand what is happening now, thank you @jedevc. Pushing the fix now.